src/Security/UserClassBuilder.php

```diff
@@ -37,7 +38,10 @@ public function addUserInterfaceImplementation(ClassSourceManipulator $manipulat
 
         $this->addPasswordImplementation($manipulator, $userClassConfig);
 
-        $this->addEraseCredentials($manipulator);
+        /* @legacy - Remove "eraseCredentials()" when Symfony 7.4 is no longer supported */
+        if(Kernel::VERSION_ID < 70300){
+            $this->addEraseCredentials($manipulator);
```

Comment:
I don't know how to deal with tests and fixtures in `tests/Security/fixtures/expected/*.php`

Can tell me what to do ?